crates/sierra-casm/src/lib.rs

```diff
@@ -0,0 +1,63 @@
+use cairo_lang_utils::bigint::BigUintAsHex;
+use num_bigint::BigUint;
+use serde::de::DeserializeOwned;
+use serde::Serialize;
+use serde_json::Error;
+use starknet::core::types::FlattenedSierraClass;
+
+use cairo_lang_starknet::casm_contract_class::CasmContractClass;
+use cairo_lang_starknet::contract_class::{ContractClass, ContractEntryPoints};
+use cairo_lang_starknet_v1_0_0_alpha6::casm_contract_class::CasmContractClass as CasmContractClassV1Alpha;
+use cairo_lang_starknet_v1_0_0_alpha6::contract_class::ContractClass as ContractClassV1Alpha;
+use cairo_lang_starknet_v1_0_0_rc0::casm_contract_class::CasmContractClass as CasmContractClassV1Rc0;
+use cairo_lang_starknet_v1_0_0_rc0::contract_class::ContractClass as ContractClassV1Rc0;
+use cairo_lang_starknet_v1_1_1::casm_contract_class::CasmContractClass as CasmContractClassV1_1;
+use cairo_lang_starknet_v1_1_1::contract_class::ContractClass as ContractClassV1_1;
+
+pub fn compile(definition: &FlattenedSierraClass) -> anyhow::Result<CasmContractClass> {
```

Comment:
I think the interface should be more universal, so that the user does not have to use `starknet::core` in order to compile